File: FRONT/AmusementPark/src/app/models/images/image-tag-dto.ts
import { LocalizedItemDto } from '../shared/localized-item-dto';

export interface ImageTagDto {
  id: string;
  slug: string;
  labels: LocalizedItemDto<string>[];
  descriptions: LocalizedItemDto<string>[];
  isActive: boolean;
  createdAt: string;
  updatedAt: string;
}

File: API/Dtos/Dtos/Images/ImageTagDto.cs
using Dtos.Shared;

namespace Dtos.Images
{
    public sealed class ImageTagDto
    {
        public string Id { get; set; } = string.Empty;
        public string Slug { get; set; } = string.Empty;
        public List<LocalizedItemDto<string>> Labels { get; set; } = new();
        public List<LocalizedItemDto<string>> Descriptions { get; set; } = new();
        public bool IsActive { get; set; }
        public DateTime CreatedAt { get; set; }
        public DateTime UpdatedAt { get; set; }
    }
}
